src/obfl-specification.html

```diff
@@ -1195,6 +1195,36 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd class="markup note">This attribute is only available in the context of a field element.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd class="markup note">These are commonly available variables. Depending
+      on the context, other variables may be available as well.</dd>
```

Comment:
I also said this when we discussed the matter earlier: I think it might be slightly clearer if all the variable documentation would be centralized here, under "evaluate", instead of having some other documentation under  "table-of-contents", "on-volume-start", "on-volume-end", "on-page-start" and "on-page-end". Firstly because this gives you a nice overview, and secondly I think it will also be easier to find (or remember where it was).

A downside would be that it can be hard to explain certain variables without also explaining the whole context, resulting in repetition. This could be solved by referring to the relevant sections in the spec when needed.

If we go for the "distributed" approach anyway I think we should at least be specific about those "other variables" that may be available as well, by listing all their names and referring to the sections where they are defined.

Reply:
On second thought the "distributed" approach is fine. But see the last sentence of my previous comment.

Reply:
I go for the distributed approach with a list of all variables at the `evaluate` element. :-)